draft-ietf-rats-corim.md

```diff
@@ -771,26 +779,24 @@ The types defined for a group identified are UUID and variable-length opaque byt
 
 ##### Measurements
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A measurement can be directly compared with attestation evidence, or it can be a quality that is not directly comparable with evidence.
+The scope of a mroperty is defined by the `environment-map` and optional `mkey` it appears with.
 
-An environment has one or more measurable elements. Each element can have a
-dedicated measurement or multiple elements could be combined into a single
-measurement. Measurements can have class, instance or group scope.  This is
-typically determined by the triple's environment.
+Measurements can be of a variety of things including software, firmware, configuration files, read-only memory, fuses, IO ring configuration, partial reconfiguration regions, etc. Measurements comprise raw values, digests, or status information.
 
-Class measurements apply generally to all the Attesters in the given class.
-Instance measurements apply to a specific Attester instance.  Environments
-identified by a class identifier have measurements that are common to the
-class. Environments identified by an instance identifier have measurements that
-are specific to that instance.
+Each element can have a dedicated measurement/endorsement value or multiple elements could be combined into a single measurement.
+An `environment-map` contains at least one of: class, instance, or group.
+The more elements in the `environment-map`, the narrower the scope, since more elements must be in common to match.
+Measurements paired with an `environment-map` that contains only a class may be referred to as class mroperties, and similarly with instance and group measurements for respective elements of the `environment-map`.
 
-The supply chain entity that is responsible for providing the the measurements (i.e. Reference Values or Endorsed Values)
-is by default the CoRIM signer. If a different entity is authorized to provide measurement values,
-the `authorized-by` statement can be supplied in the `measurement-map`.
+A measurement from an Attester is called Evidence.
+A measurement in a `reference-triple-record` is called a Reference Value.
+A measurement in another triple is called an Endorsement.
```

Comment:
Another triple?

Reply:
Every non-reference-value-triple that contains measurement-map is some flavor of endorsement, so that's the most concise way I could state that. I'm still unclear if there's a conditional relationship between reference triples and the rest of the triples in the same CoMID.

Reply:
>I'm still unclear if there's a conditional relationship between reference triples and the rest of the triples 

If they're related, the relationship is defined by the 'condition' portion of the RV triple. In the `reference-triples` case, both the `environment-map` and `measurement-map` are the condition. If there is a match, the "corroborated" values (the actual state representation) is added to the ACS under the RVP authority. If RVs are already expressed as actual state representations, then you can say "corroboration" is passthrough and the RVPs are copied into the ACS (differing only by authority from what was matched).

There is an ongoing discussion whether RVs should be used to "corroborate" ACS values that originated from Endorsers vs. Attesters. Both are representations of **actual state**. So, its technically possible. The use case for allowing it is when there are claims that can't reasonably be asserted by an AE that a vendor of that Attester wishes to supply reference values. A contrived example is if a CC certifying lab reports that an Attester meets EAL4+, but the Attester vendor wants to restrict the upper bound (for some reason), then the attester vendor could create a RV for a range, say EAL2+ to EAL4. The RV condition applies to claims that only an endorser would make. But the Verifier can easily process them just like RVs written to match Evidence.

The concern is a constrained verifier might not have resources necessary to handle a large set of inputs and one way to reduce them is to disallow processing of RVs aimed at matching Endorsements.

Reply:
> Another triple?

It would be clearer to mention a specific endorsement triple such as `endorsed-triples`